osmoutils/accum/accum.go

```diff
@@ -63,10 +63,11 @@ func setAccumulator(accum AccumulatorObject, amt sdk.DecCoins) {
 	osmoutils.MustSet(accum.store, formatAccumPrefixKey(accum.name), &newAccum)
 }
 
-// UpdateAccumulator updates the accumulator's value by amt.
-// It does so by incresing the value of the accumulator by
+// AddToAccumulator updates the accumulator's value by amt.
+// It does so by increasing the value of the accumulator by
 // the given amount. Persists to store. Mutates the receiver.
-func (accum *AccumulatorObject) UpdateAccumulator(amt sdk.DecCoins) {
+// make this safely return error when negative dec coins
```

Comment:
Is this comment a TODO?

Reply:
This was a TODO but now has been deleted